### DISCUSSION TITLE
Apple Car千呼萬喚不出來

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS3073-C/Python-Community-GS3073-C#477] [General]
感覺大家關注電動車的重點不外乎就是電池續航和自動駕駛，在這方便我還是比較看好Tesla的電動車，畢竟不管在量產技術、能源技術、AI自動駕駛技術都領先各個車廠一大截。不知大家是怎沒想的？
https://ec.ltn.com.tw/article/breakingnews/4082220

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by brian431** [comment]
雖然現在特斯拉遙遙領先其他車廠，但傳統車廠在這幾年也逐漸開始佈局電動車了，屆時應該就不會只有特斯拉佔有幾乎整個電動車市場。

**Comment by xCk27x** [comment]
各車廠都慢慢開始完善他們的電動車生態了
目前看來特斯拉的誘因逐漸減少
反倒是價格部分 其他車廠都比較友善

**Comment by BingRong1217** [comment]
Tesla確實是電動車的主流

**Comment by skyling0299** [comment]
自駕車方面多半是講求安全、便利性、以及道路預測與減少人力的傾向
但以台灣來說，自駕功能似乎並不是太便利(道路設計辨識度差、機動性車輛變因大)

**Comment by limitx0** [comment]
Apple Car的外觀和內部裝設的猜想圖都好有造型
希望快點做出來

**Comment by larry40305** [comment]
從這個角度來看，這必定是個前衛大膽的想法。自動車，發生了會如何，不發生又會如何。

**Comment by liangyinglly** [comment]
看來之後蘋果跟特斯拉會是路上主要的車種

**Comment by boson13579** [comment]
短時間而言特斯拉應該還會是電動車的領袖，因為他們掌握的技術及名聲遠遠超過其他車廠。

**Comment by 0212Jerry** [comment]
我覺得如果蘋果要自己生產汽車的話必定要從上游就開始整合，畢竟蘋果汽車一出來就肯定會是亮點，全世界都在看蘋果的產品會如何呈現。而且許多好的工廠不一定甘願給蘋果做代工。

**Comment by Vincent20030314** [comment]
特斯拉還會是主流一段時間，短時間應該不會被超越。
